Reed/Reader/views/ReaderView.swift

```diff
@@ -44,7 +44,7 @@ struct ReaderView: View {
             GeometryReader { geometry in
                 BottomSheetView (
                     isOpen: self.$isBottomSheetExpanded,
-                    maxHeight: geometry.size.height * 0.4) {
+                    maxHeight: geometry.size.height * 0.435) {
```

Comment:
Can you take a screenshot of the new margin?
Also, can you test to confirm that it looks the way we want on various iPhone devices?

Reply:
I don't think using ratios does that well so I will experiment with some constants

Reply:
constans work well